### PR TITLE
ClipboardCopy widget

### PR DESCRIPTION
**Pull request by jrusz**

### CODE REVIEW

**Comment by jrusz** [comment]
I encountered one suspicious thing. The text from the inputs was surrounded by one white space compared to the PF4 where it was not. Is that possibly a bug on the testing page or the widget itself?

File: src/widgetastic_patternfly5/components/clipboardcopy.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        super().__init__(parent, logger=logger)
+        if locator:
+            self.locator = locator
+        else:
+            self.locator = self.DEFAULT_LOCATOR
+
+
+class ClipboardCopy(BaseClipboardCopy, View):
+    ROOT = ParametrizedLocator("{@locator}")
+    DEFAULT_LOCATOR = ".//div[contains(@class,'pf-v5-c-clipboard-copy')]"
```
Comment (LightOfHeaven1994) [inline src/widgetastic_patternfly5/components/clipboardcopy.py:29]: same, remove `pf-v5`

File: src/widgetastic_patternfly5/components/clipboardcopy.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+
+from widgetastic_patternfly5 import Button
+
+
+class BaseClipboardCopy:
+    BUTTON_LOC = ".//button[contains(@class, 'pf-v5-c-button pf-m-control')]"
+    TEXT_LOC = ".//span[contains(@class, 'pf-v5-c-form-control')]/input"
```
Comment (LightOfHeaven1994) [inline src/widgetastic_patternfly5/components/clipboardcopy.py:10]: same, remove `pf-v5`

**Review by LightOfHeaven1994 — CHANGES_REQUESTED** [review]
generally lgtm, just couple small fixes needed

**Review by LightOfHeaven1994 — APPROVED** [review]
lgtm, great work!

**Pull request merged**